devtools-ci-index.yaml

```diff
@@ -619,13 +620,93 @@
             ssh_cmd="ssh $sshopts $CICO_hostname"
             $ssh_cmd yum -y install rsync
             rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload
-            $ssh_cmd -t "cd payload && /bin/bash cico_build.sh && /bin/bash cico_deploy.sh"
+            $ssh_cmd -t "cd payload && /bin/bash cico_build.sh"
             rtn_code=$?
             cico node done $CICO_ssid
             rsync -e "ssh $sshopts" -Ha --include 'target/**/target/surefire-reports/*.xml' --exclude '*' $CICO_hostname:payload $(pwd)/
             exit $rtn_code
+    publishers:
+      - junit:
+            results: target/**/target/surefire-reports/*.xml
+      - archive:
+            artifacts: che.image.env
+      - trigger:
+          project: eclipse-che-verification
+          threshold: SUCCESS
 
 - job:
+    name: 'eclipse-che-verification'
+    wrappers:
+        - che_credentials_wrapper
+    defaults: global
+    node: devtools
+    builders:
+    builders:
+      - copyartifact:
+          project: devtools-build-run-che-build-master
+          filter: "*che.image.env"
+          optional: false
+          flatten: true
+          do-not-fingerprint: true
+      - shell: |
+            ## At first, deploy Che server
+            . ./che.image.env
+            export VAULT_PORT=8200
+            export VAULT_ROOT_TOKEN_ID=${RHCHEBOT_DOCKER_HUB_PASSWORD}
+            export VAULT_HOST=li546-232.members.linode.com
+            export SECRET_KEY=os_token
+            export OSO_TOKEN=$(curl -sSL \
+                 -H "X-Vault-Token: ${VAULT_ROOT_TOKEN_ID}" \
+                 -H "Content-Type: application/json" \
+                 -X GET \
+                 http://${VAULT_HOST}:${VAULT_PORT}/v1/secret/${SECRET_KEY} | jq .data.value)
```

Comment:
Are you sure that `jq` is installed here when you run this command? If it's not you may want to use `awk` or `sed` to parse the value here